src/Record/Record.Model/Immutable/Record.cs

```diff
@@ -118,6 +140,72 @@ public IGraph MetadataGraph()
         return tempGraph;
     }
 
+    private void AssertDescribesConstraint()
+    {
+        if (!AskIfDescribedObjectExistOnContentGraph())
+            throw new RecordException("All described objects on the metadata graph must exist as subjects on the content graph.");
+
+        if (AskIfContentSubjectIsUnreachableFromMetadata())
+            throw new RecordException("All nodes on the content graph must be reachable through the describes predicate on the metadata graph.");
+    }
+
+    private bool AskIfDescribedObjectExistOnContentGraph()
+    {
+        var parameterizedQuery = new SparqlParameterizedString(@"
```

Comment:
Jeg tror det blir noe feil her. Det vi trenger å vite er at for alt som står i describes, må det finnes i en content record. Det jeg tror spørringa spør om er, finnes det minst en describes som også står i content. Så hvis det er flere describes og bare en av dem fins i innholdet vil det bli ulik oppførsel